nemo/nemo/utils/exp_logging.py

```diff
@@ -41,11 +43,11 @@ def get_logger(name):
     return loggers[name]
 
 
-def copy_wo_overwrite(dir_, file_to_copy):
+def copy_wo_overwrite(dir_, file_to_copy, tm_suf):
```

Comment:
Add description of method and its arguments.

Reply:
Function has been removed